### ISSUE TITLE
[Item] Rumsey Rum Black Label Not Stacking with Sagefish Delight

### ISSUE THREAD

**Issue by goldenpipes** [dalaranwow/dalaran-wow#4481]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1.  Title Says it all.
2.  These two items should stack buffs.  Rumsey Rum is +15 Stamina Sagefish is 8 mana per 5
3.  Eat sagefish delight, then use Rum, you will lose the well fed buff,

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Spell: Update stacking of Rumsey's alcohols 

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4481
* commit: a68689c12660397604fb9adfa2cbdfed134fda6c

By clavarnway (16,135 – 2·19·152) on 2008/02/17 (Patch 2.3.3)
"PTR 2.1 patch notes.: “Rumsey Rum”, “Rumsey Rum Light”, “Rumsey Rum Dark”, “Rumsey Rum Black Label”, “Halaani Whiskey”, “Gordok Green Grog”, “Raptor Punch”, “Stormstout”, “Trogg Ale”, and “Thunderbrew Lager”: The stat bonuses from these drinks no longer stacks with stamina bonuses from other foods."

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
